sdk/typescript/src/rpc/faucet-client.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import fetch from 'cross-fetch';
+
+import { FaucetResponse, SuiAddress } from '../types';
+import { HttpHeaders } from './client';
+
+export async function requestSuiFromFaucet(
```

Comment:
Why not just have this defined inline in the RPC client instead of in this helper function? Seems like unnecessary indirection.

Reply:
Access to faucet and fullnode are two separate concerns. I added faucet access to provider as a convenience method but I think people should still be able to access faucet without using a provider to fullnode.